### PR TITLE
sort example

### PR DESCRIPTION
**Pull request by PammyS**
For issues/1449

### CODE REVIEW

**Comment by p013570** [comment]
Looks great, thank you. Just need to fix the checkstyle warnings. Unfortunately Travis CI doesn't log the actual errors just the module that failed checkstyle, the core-rest module. So I've just checked out the code and run:

```bash
 mvn clean install -pl :core-rest -am -Pquick
 mvn verify -Panalyze -pl :core-rest
```

and it lists the errors:

```
rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/example/DefaultExamplesFactory.java:399:62: error: '{' is not preceded with whitespace.
rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/example/ExamplesFactory.java:27: error: Wrong order for 'uk.gov.gchq.gaffer.operation.impl.compare.Sort' import.
rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/example/ExamplesFactory.java:27:8: error: Unused import - uk.gov.gchq.gaffer.operation.impl.compare.Sort.
```

File: rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/example/DefaultExamplesFactory.java [hunk 1]
```diff
@@ -391,6 +395,25 @@ public GenerateElements generateElements() {
         return op;
     }
 
+    @Override
+    public OperationChain<Iterable<? extends Element>> sort(){
```
Comment (p013570) [inline rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/example/DefaultExamplesFactory.java]: Please could you update this sort method so it returns: Sort

Then the example would just be:

```java
new Sort.Builder()
    .comparators(new ElementPropertyComparator.Builder()
            .groups("entity", "edge")
            .property("count")
            .reverse(true)
            .build())
    .resultLimit(20)
    .deduplicate(true)
    .build()
```

Comment (PammyS) [reply rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/example/DefaultExamplesFactory.java]: Sure! Thanks!

Comment (PammyS) [reply rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/example/DefaultExamplesFactory.java]: @p013570 Changes done please check

**Comment by p013570** [comment]
Brilliant thanks!

**Pull request merged**